metric_learn/base_metric.py

```diff
@@ -242,6 +242,118 @@ def transform(self, X):
     """
 
 
+class BilinearMixin(BaseMetricLearner, metaclass=ABCMeta):
+  r"""Bilinear similarity learning algorithms.
+
+  Algorithm that learns a Bilinear (pseudo) similarity :math:`s_M(x, x')`,
+  defined between two column vectors :math:`x` and :math:`x'` by: :math:
+  `s_M(x, x') =  x M x'`, where :math:`M` is a learned matrix. This matrix
```

Comment:
You call the matrix W in the doc, let's stick to the same convention

Reply:
Changed to W